VAMobile/src/components/FormWrapper/FormWrapper.test.tsx

```diff
@@ -63,112 +59,85 @@ context('FormWrapper', () => {
         resetErrors={resetErrors}
         setOnSaveClicked={setOnSaveClicked}
         onSaveClicked={onSaveClicked}
       />,
     )
-
-    testInstance = component.UNSAFE_root
   }
 
   beforeEach(() => {
     initializeTestInstance()
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
-  })
-
   describe('when the picker calls setError with an empty string', () => {
-    it('should set the error to empty string', async () => {
-      await waitFor(() => {
-        let shortenedFieldsList = formFieldsList[1]
-        initializeTestInstance([shortenedFieldsList])
-        testInstance.findByType(VAModalPicker).props.setError('')
-        expect(screen.queryByText('second error message')).toBeFalsy()
-      })
+    it('should set the error to empty string', () => {
+      initializeTestInstance(pickerFieldList)
+      screen.UNSAFE_getByType(VAModalPicker).props.setError('')
+      expect(screen.queryByText('second error message')).toBeFalsy()
     })
   })
 
   describe('when the textinput calls setError with an empty string', () => {
-    it('should set the error to empty string', async () => {
-      await waitFor(() => {
-        let shortenedFieldsList = formFieldsList[0]
-        initializeTestInstance([shortenedFieldsList])
-        testInstance.findByType(VATextInput).props.setError('')
-        expect(screen.queryByText('first error message')).toBeFalsy()
-      })
+    it('should set the error to empty string', () => {
+      initializeTestInstance(textInputFieldList)
+      screen.UNSAFE_getByType(VATextInput).props.setError('')
+      expect(screen.queryByText('first error message')).toBeFalsy()
     })
   })
 
   describe('when the checkbox calls setError with an empty string', () => {
-    it('should set the error to empty string', async () => {
-      await waitFor(() => {
-        let shortenedFieldsList = formFieldsList[2]
-        initializeTestInstance([shortenedFieldsList])
-        testInstance.findByType(VASelector).props.setError('')
-        expect(screen.queryByText('third error message')).toBeFalsy()
-      })
+    it('should set the error to empty string', () => {
+      initializeTestInstance(selectorFieldList)
+      screen.UNSAFE_getByType(VASelector).props.setError('')
+      expect(screen.queryByText('third error message')).toBeFalsy()
     })
   })
 
   describe('when the picker calls setError with no parameter', () => {
-    it('should set the error to the field error message', async () => {
-      await waitFor(() => {
-        let shortenedFieldsList = formFieldsList[1]
-        initializeTestInstance([shortenedFieldsList])
-        testInstance.findByType(VAModalPicker).props.setError()
-        expect(screen.getByText('second error message')).toBeTruthy()
-      })
+    it('should set the error to the field error message', () => {
+      initializeTestInstance(pickerFieldList)
+      screen.UNSAFE_getByType(VAModalPicker).props.setError()
```

Comment:
similar to the other one, there should be another way that isn't this unsafe way to get this